tests/func/experiments/test_init.py

```diff
@@ -18,8 +23,12 @@ def test_init(tmp_dir, dvc):
     code_path = os.path.join(CmdExperimentsInit.CODE, "copy.py")
     script = f"python {code_path}"
 
+    capsys.readouterr()
     assert main(["exp", "init", script]) == 0
-    assert load_yaml(tmp_dir / "dvc.yaml") == {
+    out, err = capsys.readouterr()
```

Comment:
I was not able to test for `assert not caplog.text` because there are false issues in Macos in CI at least. `caplog.clear()` did not help for some reason.